server/controllers/message.js

```diff
@@ -0,0 +1,59 @@
+const Message = require("../models/Message");
+const Conversation = require("../models/Conversation");
+
+// @route POST /new
+// @desc create a new message associated with a conversation
+// @access Private
+exports.createNewMessage = async (req, res) => {
+  const userId = req.user.id;
+  const { conversationId, content } = req.body;
+
+  if (conversationId && content) {
+    const conversation = await Conversation.findById({ _id: conversationId });
+
+    if (!conversation) {
+      res.status(400).send("Conversation not found.");
+    } else {
+      const newMessage = await Message.create({
+        conversationId,
+        author: userId,
+        content,
+      });
+
+      if (!newMessage) {
+        res.status(500).send("An error occurred while sending a message.");
+      } else {
+        const willRead = conversation.users.filter((user) => String(user) !== String(userId));
+
+        await Conversation.findByIdAndUpdate(
```

Comment:
```suggestion
        conversation.save();
```
Same as comment above!